Tella/Data/Database/TellaDataBase.swift

```diff
@@ -26,14 +26,18 @@ class TellaDataBase : DataBase {
             case 0:
                 createTables()
             case 1:
+                alterTable()
+                createTables()
+                renameUpdatedDateColumn()
+            case 2:
```

Comment:
In this version we need to add only the Template table and Uwazi server

Reply:
func checkVersions() {
        do {
            let oldVersion = try statementBuilder.getCurrentDatabaseVersion()
            
            switch oldVersion {
            case 0:
                createTables()
            case 1:
                createFeedbackTable()
                renameUpdatedDateColumn()
            case 2:
                createTemplateTableForUwazi()
                createUwaziServerTable()
            default :
                break
            }
            
            try statementBuilder.setNewDatabaseVersion(version: D.databaseVersion)
            
        } catch let error {
            debugLog(error)
        }
    }